# Households comparison - 2060

Pairwise comparison of total households at **2060** between forecasts, by control/target area and grouped by county. Each section shows two tables, both against the same baseline:

- **A vs B**: Vision 2060 Scaled vs Vision 2060
- **C vs B**: Vision 2060 Scaled (County) vs Vision 2060

`% Diff = (forecast - baseline) / baseline * 100`.

In [1]:
from comparison_helpers import compare_hh_target_area, compare_hh_region

FORECAST_A = 'vision_2060_scaled'
FORECAST_B = 'vision_2060'  # baseline (denominator)
FORECAST_C = 'vision_2060_scaled_by_county'
YEAR = '2060'

## Region

In [2]:
compare_hh_region(FORECAST_A, FORECAST_B, YEAR)

,Vision 2060 Scaled,Vision 2060,Diff,% Diff
King,"1,264,732","1,290,051","-25,319",-2.0%
Pierce,"489,680","501,408","-11,728",-2.3%
Snohomish,"549,089","567,836","-18,747",-3.3%
Kitsap,"146,426","149,642","-3,217",-2.1%
Region,"2,449,926","2,508,937","-59,011",-2.4%


In [3]:
compare_hh_region(FORECAST_C, FORECAST_B, YEAR)

,Vision 2060 Scaled by County,Vision 2060,Diff,% Diff
King,"1,383,291","1,290,051","+93,240",+7.2%
Pierce,"454,969","501,408","-46,439",-9.3%
Snohomish,"483,252","567,836","-84,584",-14.9%
Kitsap,"128,415","149,642","-21,227",-14.2%
Region,"2,449,927","2,508,937","-59,010",-2.4%


## King County

In [4]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53033)

,Vision 2060 Scaled,Vision 2060,Diff,% Diff
Bellevue,"109,330","113,184","-3,855",-3.4%
Seattle,"438,696","444,334","-5,638",-1.3%
Auburn,"42,915","44,163","-1,248",-2.8%
Bothell,"19,706","20,319",-613,-3.0%
Burien,"30,596","31,453",-857,-2.7%
Federal Way,"51,543","52,786","-1,244",-2.4%
Issaquah,"19,261","19,478",-217,-1.1%
Kent,"59,045","59,993",-948,-1.6%
Kirkland,"52,224","53,243","-1,018",-1.9%
Redmond,"52,398","54,092","-1,694",-3.1%


In [5]:
compare_hh_target_area(FORECAST_C, FORECAST_B, YEAR, 53033)

,Vision 2060 Scaled by County,Vision 2060,Diff,% Diff
Bellevue,"127,379","113,184","+14,194",+12.5%
Seattle,"465,096","444,334","+20,762",+4.7%
Auburn,"48,759","44,163","+4,596",+10.4%
Bothell,"22,578","20,319","+2,259",+11.1%
Burien,"34,609","31,453","+3,156",+10.0%
Federal Way,"57,366","52,786","+4,580",+8.7%
Issaquah,"20,278","19,478",+799,+4.1%
Kent,"63,482","59,993","+3,489",+5.8%
Kirkland,"56,993","53,243","+3,750",+7.0%
Redmond,"60,329","54,092","+6,237",+11.5%


## Pierce County

In [6]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53053)

,Vision 2060 Scaled,Vision 2060,Diff,% Diff
Tacoma,"146,687","151,368","-4,681",-3.1%
Auburn,"3,741","3,745",-4,-0.1%
Lakewood,"39,378","40,501","-1,123",-2.8%
Puyallup,"28,067","28,955",-888,-3.1%
University Place,"21,699","22,304",-605,-2.7%
DuPont,"6,867","7,129",-262,-3.7%
Fife,"6,921","7,144",-223,-3.1%
Fircrest,"3,954","4,042",-88,-2.2%
"Mid-County, Parkland-Spanaway-Midland, South Hill","81,831","83,794","-1,963",-2.3%
Sumner,"7,292","7,531",-239,-3.2%


In [7]:
compare_hh_target_area(FORECAST_C, FORECAST_B, YEAR, 53053)

,Vision 2060 Scaled by County,Vision 2060,Diff,% Diff
Tacoma,"132,832","151,368","-18,535",-12.2%
Auburn,"3,730","3,745",-15,-0.4%
Lakewood,"36,054","40,501","-4,447",-11.0%
Puyallup,"25,439","28,955","-3,516",-12.1%
University Place,"19,909","22,304","-2,395",-10.7%
DuPont,"6,091","7,129","-1,038",-14.6%
Fife,"6,260","7,144",-884,-12.4%
Fircrest,"3,693","4,042",-349,-8.6%
"Mid-County, Parkland-Spanaway-Midland, South Hill","76,023","83,794","-7,771",-9.3%
Sumner,"6,584","7,531",-946,-12.6%


## Snohomish County

In [8]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53061)

,Vision 2060 Scaled,Vision 2060,Diff,% Diff
Everett,"101,057","105,534","-4,476",-4.2%
Bothell,"17,119","17,897",-778,-4.3%
Lynnwood,"35,411","36,978","-1,567",-4.2%
Arlington,"19,048","19,928",-881,-4.4%
Bothell MUGA,"19,668","20,286",-617,-3.0%
Edmonds,"31,960","33,048","-1,088",-3.3%
Edmonds MUGA,"2,306","2,363",-57,-2.4%
Everett MUGA,"30,573","31,648","-1,075",-3.4%
Larch Way Overlap,"5,954","6,287",-333,-5.3%
Lynnwood MUGA,"29,807","31,061","-1,255",-4.0%


In [9]:
compare_hh_target_area(FORECAST_C, FORECAST_B, YEAR, 53061)

,Vision 2060 Scaled by County,Vision 2060,Diff,% Diff
Everett,"85,338","105,534","-20,195",-19.1%
Bothell,"14,387","17,897","-3,510",-19.6%
Lynnwood,"29,907","36,978","-7,071",-19.1%
Arlington,"15,955","19,928","-3,973",-19.9%
Bothell MUGA,"17,500","20,286","-2,786",-13.7%
Edmonds,"28,138","33,048","-4,911",-14.9%
Edmonds MUGA,"2,105","2,363",-259,-10.9%
Everett MUGA,"26,800","31,648","-4,849",-15.3%
Larch Way Overlap,"4,786","6,287","-1,501",-23.9%
Lynnwood MUGA,"25,401","31,061","-5,660",-18.2%


## Kitsap County

In [10]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53035)

,Vision 2060 Scaled,Vision 2060,Diff,% Diff
Bremerton,"27,728","28,608",-880,-3.1%
Bremerton UGA,"5,762","5,894",-132,-2.2%
Silverdale,"14,901","15,461",-560,-3.6%
Bainbridge Island,"13,246","13,494",-247,-1.8%
Kingston UGA,"3,342","3,520",-178,-5.1%
Port Orchard,"12,188","12,650",-463,-3.7%
Port Orchard UGA,"7,749","7,899",-151,-1.9%
Poulsbo,"7,235","7,411",-177,-2.4%
Poulsbo PUTA,766,813,-47,-5.7%
Central Kitsap UGA,"12,726","13,051",-325,-2.5%


In [11]:
compare_hh_target_area(FORECAST_C, FORECAST_B, YEAR, 53035)

,Vision 2060 Scaled by County,Vision 2060,Diff,% Diff
Bremerton,"22,800","28,608","-5,808",-20.3%
Bremerton UGA,"5,026","5,894",-868,-14.7%
Silverdale,"11,767","15,461","-3,694",-23.9%
Bainbridge Island,"11,862","13,494","-1,632",-12.1%
Kingston UGA,"2,342","3,520","-1,178",-33.5%
Port Orchard,"9,597","12,650","-3,054",-24.1%
Port Orchard UGA,"6,904","7,899",-995,-12.6%
Poulsbo,"6,246","7,411","-1,166",-15.7%
Poulsbo PUTA,506,813,-307,-37.8%
Central Kitsap UGA,"10,908","13,051","-2,143",-16.4%


## Notes

- `% Diff` is computed as `(A - B) / B * 100`, where **B** (baseline) is the
  second forecast listed at the top of the notebook.
- Cell color: red when `|% Diff| >= 5%`, amber when `>= 2.5%`, green otherwise.
- Target areas present in only one forecast still appear, with `-` for the
  missing forecast's value.